src/SprykerEco/Zed/Heidelpay/Business/Mapper/QuoteToHeidelpayRequest.php

```diff
@@ -85,6 +88,47 @@ protected function mapQuoteInformation(QuoteTransfer $quoteTransfer, HeidelpayRe
         return $heidelpayRequestTransfer;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\HeidelpayRequestTransfer $heidelpayRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\HeidelpayRequestTransfer
+     */
+    protected function mapCustoemrInformation(QuoteTransfer $quoteTransfer, HeidelpayRequestTransfer $heidelpayRequestTransfer): HeidelpayRequestTransfer
+    {
+        $customerRegistrationDate = $this->findCustomerRegistrationDate($quoteTransfer->getCustomer());
+
+        $heidelpayRequestTransfer->setRiskInformation(
+            (new HeidelpayRiskInformationTransfer())
+                ->setIsCustomerGuest((bool)$quoteTransfer->getCustomer()->getIsGuest())
+                ->setCustomerSince($customerRegistrationDate)
+                ->setCustomerId($quoteTransfer->getCustomer()->getIdCustomer())
+        );
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\CustomerTransfer $customerTransfer
+     *
+     * @return string|null
+     */
+    protected function findCustomerRegistrationDate(CustomerTransfer $customerTransfer): ?string
+    {
+        $createdAtDate = $customerTransfer->getCreatedAt();
+        $createdAtDateFormatted = $createdAtDate ? $this->formatDate($createdAtDate) : $createdAtDate;
+
+        return $createdAtDateFormatted;
+    }
+
+    /**
+     * @param string $date
+     *
+     * @return string
+     */
+    protected function formatDate(string $date): string
+    {
+        return (new DateTime($date))->format('Y-m-d');
```

Comment:
It seems like it'll fail. You don't have it in use statements and PHP will try to find it under the current namespace